agent/provider/src/provider_agent.rs

```diff
@@ -278,6 +277,25 @@ impl ProviderAgent {
         })
     }
 
+    fn create_service_info(
+        inf_node_info: InfNodeInfo,
+        exeunit_desc: ExeUnitDesc,
+        offer: &OfferTemplate,
+    ) -> anyhow::Result<ServiceInfo> {
+        let exeunit_desc = exeunit_desc.build();
+        let support_payload_manifest = match offer.property(manifest::CAPABILITIES_PROPERTY) {
+            Some(value) => {
+                serde_json::from_value(value.clone()).map(|capabilities: Vec<Feature>| {
+                    capabilities.contains(&Feature::ManifestSupport)
+                })?
```

Comment:
It will NOT fail on missing `manifest::CAPABILITIES_PROPERTY` BUT it will fail on an unknown `Feature`.
Runtimes and Yagna daemons are distributed together, but I am not sure it should fail or just ignore and log it as warn.